package.json

```diff
@@ -43,7 +43,7 @@
     "wcag": "test-storybook",
     "test:unit": "jest --coverage --coverageDirectory=coverage/unit",
     "test:visual": "./tests/scripts/visual-coverage.sh",
-    "test": "./tests/scripts/combined-coverage.sh"
+    "test": "unset NODE_OPTIONS && export NODE_OPTIONS=--openssl-legacy-provider && ./tests/scripts/combined-coverage.sh"
```

Comment:
what is the issue with node 18?

Reply:
What command in the script has the issue?

Reply:
The command that has issue is `npm run test:visual`. So for test 18x in the past pull requests, the visual coverage test has always been skipped due to storybook not starting. It is due to the webpack dependency storybook uses not not supporting with node version 17 and up
<img width="530" alt="Screen Shot 2022-08-12 at 3 15 38 PM" src="https://user-images.githubusercontent.com/65748924/184428426-46a5b7a4-16ee-4520-9cb6-a5d5343c64a1.png">
.